### PR TITLE
Add tests for pyproject with multiple readme files

### PR DESCRIPTION
**Pull request by wagnerluis1982**
# Pull Request Check List

**tests/masonry/builders/test_editable_builder.py**
- Checks if content of all readme files are printed to the metadata

**tests/console/commands/test_build.py** (new file)
- Checks sdist file exists and is not empty
- Checks wheel file exists and is not empty
- Checks inside sdist tar file if contains all readme files

Closes: #6633
Depends on: python-poetry/poetry-core#551

- This PR only add **tests**

### CODE REVIEW

**Comment by wagnerluis1982** [comment]
Question to maintainers, does poetry have any integration test for `poetry build` command? I was not able to found any.

**Comment by radoering** [comment]
Probably not. (It should be located at `tests/console/commands`.)

Since the test in this PR is passing even without the poetry-core fix, I assume an integration test is required to provoke the issue?

**Comment by wagnerluis1982** [comment]
@radoering Exactly, that's why I was looking for an integration test for `build`.

I will create a new test module then.

**Comment by wagnerluis1982** [comment]
Just added a test to verify `poetry build` command.

NOTE: the pipeline is going to fail until python-poetry/poetry-core#486 is merged.

**Comment by wagnerluis1982** [comment]
Hi @radoering, after you merged python-poetry/poetry-core#486, I expected the tests to not fail (I just rebased after `master`).

I expected the pipeline to use latest poetry-core version. Does that happen?

**Comment by radoering** [comment]
No, it just uses the locked version of poetry-core (no magic). Normally, we just update the locked version after a core release.

**Comment by wagnerluis1982** [comment]
> No, it just uses the locked version of poetry-core (no magic). Normally, we just update the locked version after a core release.

No magic 🙁

In that case, it would be advisable to reopen issue #6633 to signalize users, IMHO.

**Comment by wagnerluis1982** [comment]
@Secrus the tests here are going to fail while next poetry-core isn't released.

**Comment by wagnerluis1982** [comment]
@radoering when you have a chance, please check this.

Now that poetry-core 1.5.0 is locked, the tests are passing.

File: tests/masonry/builders/test_editable_builder.py
Comment (radoering) [inline tests/masonry/builders/test_editable_builder.py:334]:
```suggestion
    assert dist_info.joinpath("METADATA").read_text(encoding="utf-8") == metadata
```
We prefer actual-expected order.

File: tests/fixtures/with_multiple_readme_files/pyproject.toml
Comment (radoering) [inline tests/fixtures/with_multiple_readme_files/pyproject.toml:6]: Can you replace author information by Sébastiens or some synthetic data, please? We want to avoid introducing any new real-world information in test data.

**Review by radoering — CHANGES_REQUESTED** [review]
Just some minor nitpicks.

File: tests/masonry/builders/test_editable_builder.py
Comment (wagnerluis1982) [reply tests/masonry/builders/test_editable_builder.py]: Got it. Done :slightly_smiling_face:

File: tests/fixtures/with_multiple_readme_files/pyproject.toml
Comment (wagnerluis1982) [reply tests/fixtures/with_multiple_readme_files/pyproject.toml]: Makes sense, I liked this.

Done!

**Review by radoering — APPROVED** [review]

**Pull request merged**
